src/poetry/packages/__init__.py

```diff
@@ -1,3 +1,5 @@
 from poetry.packages.dependency_package import DependencyPackage
 from poetry.packages.locker import Locker
-from poetry.packages.package_collection import PackageCollection
```

Comment:
This is the only import statement that I've removed manually because if we add it to `__all__` (export it), it wouldn't be detected by Pycln.

Reply:
can you clarify this for me? I would have expected this to just be added to `__all__` like the other changes. I'm not seeing any issues with it locally (perhaps it was just misspelled it?)

Reply:
It seems like it's not used anywhere in the project. Could you please double check?